### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#346] [Announcements/公告]
**Full Changelog**: https://github.com/constanline/XQuickEnergy/compare/v1.1.9...v1.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.2.0'>v1.2.0</a>.</em>

**Comment by 244276041** [comment]
绿色经营不收记录绿色金融行为（里面的任意3次）
不收自己的金币
不收好友的金币
和森林类似

**Reply by xieshensai** [reply]
小鸡自动游戏失效，如果可以增加雇佣小鸡功能更好，谢谢

**Comment by whi-tesheep** [comment]
建议复活好友能量加个对能量多少的判断，例如超过100的提醒复活。有时提醒复活只有个位数能量～尬

**Comment by echo-fannnnnn** [comment]
快快更新吧！作者辛苦啦

**Comment by yiwong412** [comment]
好像用不了了

**Comment by 1505194659** [comment]
用不了了呀作者
